data/scene/lodglobes/earth/earth.mod

```diff
@@ -63,7 +64,7 @@ return {
         Renderable = {
             Type = "RenderableGlobe",
             Radii = earthEllipsoid,
-            SegmentsPerPatch = 64,
+            SegmentsPerPatch = 128,      
```

Comment:
We changed it together when we were generating the Earth to Mars video. When one increases the number of segments per patch the effects of the normals on the atmosphere when the camera is in space is less noticeable.